### ISSUE TITLE
555388C8 - Rayman Legends Demo

### ISSUE THREAD

**Issue by Gh0stblade** [xenia-project/game-compatibility#41]
- Marketplace Link: http://marketplace.xbox.com/en-GB/Product/Rayman-Legends-Demo/66acd000-77fe-1000-9115-d802555388c8

- Tested with: https://github.com/benvanik/xenia/commit/6d4d1db3106028cce6e28b5cb099324e146a7a25

- Ubisoft logo video is purple and completely corrupt displaying garbage. Hangs completely after about 5 seconds and nothing happens.

**Comment by Gh0stblade** [comment]
Updating this

-Tested with:
https://github.com/benvanik/xenia/commit/b663b615bfde8b2d5568802e0e52064696316fd5

- Ubisoft logo video is purple and completely corrupt displaying garbage. Hangs completely after about 5 seconds and nothing happens.
- Goes to the title screen then doesn't progress after the "do not turn off when saving" message pops up.

Suggested tags: video-garbage, gpu-corrupt-drawing.  state-menus.

**Comment by Kattus** [comment]
I've been testing this on benvanik/xenia@82ddd0430fe3792e8ce6e23dea161e5832b4f5f8 and after waiting for a few mins, I actually get into gameplay, everything is black however unless I open the menu.

**Comment by makayama** [comment]
Yes these game work fine i using full version for that , but the only problem  i face it is in saving game also note that anyone can try it to give his rating for me itsi   green status mean  playable. Thank for those who work hard to make this done .

**Comment by Kattus** [comment]
This should be closed as of #464

**Comment by Razzile** [comment]
yea why not

**Comment by gibbed** [comment]
They are separate titles/code, so it's fine to keep this open.

**Comment by Razzile** [comment]
can somebody retest this then please so the labels can be updated

**Comment by Kattus** [comment]
Wasn't sure if we separated demos and full verisons or how we did, but alright. I've tested the demo on benvanik/xenia@e95a08fc87a93567eaacf7175475fa921c9bc2d5 and it crashes right away. [Log file](https://gist.github.com/Kattus/dbf45b55fb6e8a4b78d71e307738ad64)
state-crash

**Comment by gibbed** [comment]
@Kattus If it has a distinct title ID, then it's fine. If the code between different versions of a game is actually the same (or similar enough), then we could merge them into one, but for now separate issues is fine.

**Comment by makayama** [comment]
as i said before  i testing these game in full version it is playable but audio not work and save also ..

**Comment by bdeimos** [comment]
Tested on (https://github.com/xenia-project/xenia/commit/f89b4626a37dd4d1e5a817909252d3d4dd054299)

Playable with some performance issues and small visual glitches.

(state-gameplay)

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/2151795/xenia.zip)

![untitled](https://user-images.githubusercontent.com/40533288/42126121-3a51cb16-7c8b-11e8-984d-0413afe12bd8.jpg)

**Comment by TGLTClutch** [comment]
someone update the labels^
